source/ControllableHero.hx

```diff
@@ -0,0 +1,54 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+
+using flixel.util.FlxSpriteUtil;
+
+// Part 2: Added a ControllableHero Class
+class ControllableHero extends FlxSprite
+{
+	// Movement variable
+	var speed = 10; // Couldn't figure out a better way to handle this?
+
+	// Implement Constructor
+	public function new()
+	{
+		// Invoke parent class constructor
+		super(x, y);
+		this.x = x;
+		this.y = y;
```

Comment:
these lines are unnecessary. `x` and `y` are properties defined in the `FlxObject` class that this (indirectly) inherits from - when you call `super(x, y)`, it's executing our parent class's logic, which is what ultimately ends up setting our `x` and `y` initial values.

Also, the `this` keyword is generally not necessary - the only `x` and `y` variables that are available in this method (since you haven't declared any local variables) are the instance variables attached to this class - `this.x` and `x` are both just referencing the same thing.